jaffa-components/source/java/org/apache/log4j/jdbcplus/JDBCAppenderConfig.java

```diff
@@ -0,0 +1,103 @@
+package org.apache.log4j.jdbcplus;
+
+import java.net.URL;
+
+import javax.annotation.PostConstruct;
+import javax.annotation.PreDestroy;
+
+import org.apache.log4j.BasicConfigurator;
+import org.apache.log4j.LogManager;
+import org.apache.log4j.Logger;
+import org.apache.log4j.xml.DOMConfigurator;
+import org.jaffa.config.Config;
+import org.jaffa.util.URLHelper;
+import org.springframework.beans.BeansException;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.ApplicationContextAware;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * 
+ * This class initialise the Log4j.
+ * 
+ * This should be either imported into spring root @Configuration
+ * e.g. @Import({ JDBCAppenderConfig.class)
+ * 
+ * or defined in web.xml
+ * 
+ * e.g.
+ * <context-param> <param-name>contextConfigLocation</param-name>
+ *   <param-value>org.apache.log4j.jdbcplus.JDBCAppenderConfig</param-value>
+ * </context-param>
+ * 
+ */
+@Configuration
+public class JDBCAppenderConfig implements ApplicationContextAware{
```

Comment:
I dont think it needs to be ApplicationContextAware since its already SpringConfig